### ISSUE TITLE
Cannot find module '../build/better_sqlite3.node'

### ISSUE THREAD

**Issue by JockeRider199** [TrueXPixels/quick.db#140]
I installed quick.db on Windows. 
I tried to add it to my code and when I launch my Client, I see that error : 
```
Error: Cannot find module '../build/better_sqlite3.node'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (d:\Developpement\node_modules\better-sqlite3\lib\database.js:5:21)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
```

Please can you help me ?

**Comment by TrentonGage11** [comment]
i am having the same problem, no clue how to fix it tho

**Comment by chroventer** [comment]
@JockeRider199, please join the support Discord server for the answer. Furthermore, the error occurs not because of Quick.db. Instead, because of better-sqlite3.
Thus, you can submit an issue on their [Issues page](https://github.com/JoshuaWise/better-sqlite3/issues)

**Comment by chroventer** [comment]
Please close this issue!

**Comment by TrueXPixels** [comment]
Most likely fixed with #160
